final_task/rss_reader/exceptions.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+
+class GettingRSSException(Exception):
+    def __init__(self, msg):
+        super().__init__('Problems with getting RSS: ' + msg)
```

Comment:
You don't really need to redefine a `__init__` here. You can just leave `pass` here, and this will still work

Reply:
fixed